appendix-basic65.tex

```diff
@@ -8398,10 +8521,10 @@ \subsection{RND}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$BB
 \item [Format:] {\bf RND(}type{\bf)}
-\item [Usage:] Returns a pseudo random number.
+\item [Usage:] Generates a new pseudo random number.
```

Comment:
Feel like you've avoided use of a starting verb up till now, so maybe this needs to follow suit, but could also tie in with the possibility of replacing '[Usage:]' with '[Returns:]' for all functions.

Reply:
Updated. Also tidied up the explanation slightly.